apps/web/src/data/ecosystem.json

```diff
@@ -17,6 +17,60 @@
     ],
     "imageUrl": "/images/partners/envio.png"
   },
+  {
+    "name": "ZKP2P",
+    "url": "https://zkp2p.xyz/",
+    "description": "ZKP2P is a trustless P2P fiat to crypto on/offramp powered by zero knowledge proofs. Convert USD in Venmo to USDC in 60 seconds!",
+    "tags": [
+      "onramp"
+    ],
+    "imageUrl": "/images/partners/zkp2p.png"
+  },
+  {
+    "name": "Rubic",
+    "url": "https://rubic.exchange/",
+    "description": "Rubic is a Cross-Chain Tech Aggregator for users + tools for dApps. Trade 15,500+ tokens across 70+ networks, through the aggregation of 220+ DEXs and bridges!",
+    "tags": [
+      "bridge"
+    ],
+    "imageUrl": "/images/partners/rubic.png"
+  },
+  {
+    "name": "Pockie",
+    "url": "https://www.pockie.io/",
+    "description": "Pockie is a Web 3 wallet focused on the user experience. Users can manage all tokens across the chains in one screen and customize profile with NFTs.",
+    "tags": [
+      "wallet"
+    ],
+    "imageUrl": "/images/partners/pockie.png"
+  },
+  {
+    "name": "trustalabs",
+    "url": "https://www.trustalabs.ai/",
+    "description": "Trusta Labs aims to build web3 security and data infrastructure to bring more trust and less friction to the web3 world."",
```

Comment:
Extra quotation mark here

Reply:
Good eye! Thank you so much, I've fixed the error